proto/messages.proto

```diff
@@ -263,6 +263,7 @@ enum EncryptionAlgorithm {
 }
 
 enum LegalHoldStatus {
+  UNKNOWN = 0;
   DISABLED = 0;
```

Comment:
Shouldn't this be `1`? Currently `UNKNOWN` and `DISABLED ` have the same value

Reply:
Already fixed ;)

Reply:
My bad. Update is there!